scale-tests/setup_streaming.py

```diff
@@ -153,13 +153,16 @@ def get_expected_task_count(service_options: dict) -> int:
                                         zookeeper_service["service"]["name"],
                                         "endpoint clientport", json=True)["dns"]
 
+        expected_count = get_expected_task_count
+
         service_options = {
-            "kafka": {
-                "kafka_zookeeper_uri": ",".join(zookeeper_dns)
+            "brokers": {
+                "kafka_zookeeper_uri": ",".join(zookeeper_dns),
+                "count": expected_count
```

Comment:
I don't get this line. `get_expected_task_count` is a function, therefore `expected_count` is a function too. Why are we setting the value of `count` to a function here?

Reply:
@mpereira you're right, `count` should be under `brokers` separately: https://github.com/mesosphere/dcos-commons/blob/master/frameworks/kafka/universe/config.json#L280